en/docs/learn/analytics/configuring-apim-analytics.md

```diff
@@ -147,10 +147,18 @@ The server URL of the remote WSO2 API-M Analytics server used to collect statist
 <p class="admonition-title">From DAS to SI</p>
 <p>
 Previously, when WSO2 API-M Analytics was powered by WSO2 Data Analytics Server 3.1.0, you had to add a comma to separate the DAS URLs so that the analytics events were published to both nodes. 
-However, when using WSO2 API-M Analytics 3.0.0, which is based on WSO2 Enterprise Integrator 7.0.x - Streaming Integrator, when publishing to an HA setup of APIM analytics, you need to separate the Receiver URLs by the pipe symbol (|) because the analytics events are published in a failover manner where only one node handles the processing at any given time.
+However, when using WSO2 API-M Analytics 3.0.0, which is based on WSO2 Enterprise Integrator 7.0.x - Streaming Integrator, when publishing to a minimum HA setup of APIM analytics, you need to separate the Receiver URLs by using the pipe symbol (|) because the analytics events are published in a failover manner where only one node handles the processing at any given time.
```

Comment:
@RAVEENSR, please confirm the following part of this paragraph.

"WSO2 API-M Analytics 3.0.0, which is based on WSO2 Enterprise Integrator 7.0.x - Streaming Integrator"

Reply:
We should avoid mentioning that API Manager Analytics 3.x.x(any of the 3.x.x. version) is based on Streaming Integrator. API Manager Analytics is completely unique and it is not something like API Manager Analytics 2.6.0, which was built on top of Stream Processor 4.3 version. Hence, we should remove the `which is based on WSO2 Enterprise Integrator 7.0.x - Streaming Integrator` section.